eng/pipelines/libraries/run-test-job.yml

```diff
@@ -81,6 +85,16 @@ jobs:
                     /bl:$(Build.SourcesDirectory)/artifacts/log/$(_BuildConfig)/overrideCoreclrFromLiveDrop.binlog
             displayName: Override $(runtimeFlavorName) from live drop
 
+        - ${{ if ne(parameters.liveRuntimeBuildConfig, '') }}:
+          - script: $(_msbuildCommand) src/libraries/pretest.proj 
+                    $(_runtimeArtifactsPathArg)
+                    $(_runtimeOSArg)
+                    /p:RuntimeFlavor=${{ parameters.runtimeFlavor }}
+                    /p:TargetArchitecture=${{ parameters.archType }}
+                    /p:Configuration=${{ parameters.buildConfig }}
+                    /bl:$(Build.SourcesDirectory)/artifacts/log/$(_BuildConfig)/overrideCoreclrFromLiveDrop_pretest.binlog
```

Comment:
the binarylog should be named different. This is not overridingCoreclr, this is just running pretest. Change to `pretest.binlog` only.

Reply:
fixed